tests/unit/button/button_core_deprecated.js

```diff
@@ -0,0 +1,30 @@
+/*
+ * button_core.js
+ */
+
+
+(function($) {
+
+module( "Button: core deprecated" );
+
+test( "Calling button on a checkbox input calls checkboxradio widget", function(){
+	var checkbox = $( "#checkbox01" );
+
+	expect( 2 );
+	checkbox.button();
+
+	equal( checkbox.is( ":ui-checkboxradio" ), true,
```

Comment:
Can use `ok()` for boolean assertions, it'll output the actual value either way (thanks to a more recent improvement in QUnit).